src/components/SummaryBox/SummaryBox.stories.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+import { SummaryBox } from './SummaryBox'
+
+export default {
+  title: 'Components/SummaryBox',
+  component: SummaryBox,
+  parameters: {
+    docs: {
+      description: {
+        component: `
+### USWDS 2.0 SummaryBox component
+
+Source: https://designsystem.digital.gov/components/summary-box
+`,
+      },
+    },
+  },
+}
+
+const summaryBoxContent = (
+  <div className="usa-summary-box__text">
```

Comment:
It looks like this `<div className="usa-summary-box__text">` is repeated in the Story (since it's also rendered in the component itself) - can we remove?